src/ApparentHorizons/StrahlkorperDataBox.hpp

```diff
@@ -0,0 +1,207 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "ApparentHorizons/Strahlkorper.hpp"
+#include "DataStructures/DataBox.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// Holds tags and ComputeItems associated with a Strahlkorper.
+namespace StrahlkorperDb {
+
+// This struct supplies shorter names for longer types used below.
+template <typename Frame>
+struct types {
+  using ThetaPhi = tnsr::i<DataVector, 2, ::Frame::Spherical<Frame>>;
+  using OneForm = tnsr::i<DataVector, 3, Frame>;
+  using Vector = tnsr::I<DataVector, 3, Frame>;
+  using Jacobian =
+      Tensor<DataVector, tmpl::integral_list<std::int32_t, 2, 1>,
+             index_list<SpatialIndex<3, UpLo::Up, Frame>,
+                        SpatialIndex<2, UpLo::Lo, ::Frame::Spherical<Frame>>>>;
+  using InvJacobian =
+      Tensor<DataVector, tmpl::integral_list<std::int32_t, 2, 1>,
+             index_list<SpatialIndex<2, UpLo::Up, ::Frame::Spherical<Frame>>,
+                        SpatialIndex<3, UpLo::Lo, Frame>>>;
+  using InvHessian =
+      Tensor<DataVector, tmpl::integral_list<std::int32_t, 3, 2, 1>,
+             index_list<SpatialIndex<2, UpLo::Up, ::Frame::Spherical<Frame>>,
+                        SpatialIndex<3, UpLo::Lo, Frame>,
+                        SpatialIndex<3, UpLo::Lo, Frame>>>;
+  using SecondDeriv = tnsr::ii<DataVector, 3, Frame>;
+};
+
+template <typename Frame>
+struct Strahlkorper : db::DataBoxTag {
+  static constexpr db::DataBoxString label = "Strahlkorper";
+  using type = ::Strahlkorper<Frame>;
+};
+
+/// \f$(\theta,\phi)\f$ on the grid.
+/// Doesn't depend on the shape of the surface.
+template <typename Frame>
+struct ThetaPhi : db::ComputeItemTag {
+  static constexpr db::DataBoxString label = "ThetaPhi";
+  static typename types<Frame>::ThetaPhi function(
+      const ::Strahlkorper<Frame>& strahlkorper) noexcept;
+  using argument_tags = typelist<Strahlkorper<Frame>>;
+};
+
+/// \f$x_i/\sqrt{x^2+y^2+z^2}\f$ on the grid.
+/// Doesn't depend on the shape of the surface.
+template <typename Frame>
+struct Rhat : db::ComputeItemTag {
+  static constexpr db::DataBoxString label = "Rhat";
+  static typename types<Frame>::OneForm function(
+      const typename types<Frame>::ThetaPhi& theta_phi) noexcept;
+  using argument_tags = typelist<ThetaPhi<Frame>>;
+};
+
+/// jacobian(i,j) is \f$\frac{1}{r}\partial x^i/\partial\theta\f$,
```

Comment:
Please look at the rendering of the documentation, there are multiple errors.  Please place things like `jacobian` 
in ticks